e2e/init.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import detox from 'detox';
+import adapter from 'detox/runners/jest/adapter';
+
+import { detox as config } from '../package.json';
+
+jest.setTimeout(120000);
```

Comment:
Does that mean that any e2e test must be 2 minutes long max?

Reply:
Probably yes, lets increase it when needed, I have not added any tests yet, other than a proof of concept. Also, it is not running on the CI so.

Reply:
Sounds good to me!